snippets/server-apis/python/use.md

```diff
@@ -2,17 +2,30 @@
 # Controllers API
 
 # This doesn't need authentication
-@app.route("/api/public")
-@cross_origin(headers=['Content-Type', 'Authorization'])
+@APP.route("/api/public")
+@cross_origin(headers=["Content-Type", "Authorization"])
 def public():
     response = "Hello from a public endpoint! You don't need to be authenticated to see this."
     return jsonify(message=response)
 
 # This does need authentication
-@app.route("/api/private")
-@cross_origin(headers=['Content-Type', 'Authorization'])
+@APP.route("/api/private")
+@cross_origin(headers=["Content-Type", "Authorization"])
 @requires_auth
 def private():
     response = "Hello from a private endpoint! You need to be authenticated to see this."
     return jsonify(message=response)
+
+# This does need authorization
```

Comment:
Would could just be "This needs authorization".  I immediately saw "doesn't" when reading FWIW.

Reply:
Absolutely.